lib/screens/enquiry_information/ui/enquiry_information_page.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_poc/screens/enquiry_information/controller/enquiry_information_controller.dart';
+import 'package:flutter_poc/utils/constants.dart';
+import 'package:flutter_poc/utils/ui/image_slider/simple_slider.dart';
+
+class EnquiryInformation extends StatelessWidget {
+  EnquiryInformation({Key? key}) : super(key: key);
+
+  final enquiryController = EnquiryInfoController();
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        body: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            const SizedBox(height: 48),
+            Center(child: Image.asset("images/logo_name_blue.png")),
+            Expanded(
+                flex: 3,
+                child: Container(
+                  alignment: Alignment.center,
+                  child: ImageSliderWidget(
+                    imageUrls: enquiryController.slideImages,
+                    imageBorderRadius: BorderRadius.circular(8.0),
+                    imageHeight: MediaQuery.of(context).size.height/3.5,
+                  ),
+                )),
+            Padding(
+              padding: const EdgeInsets.only(left: 25, right: 25,bottom: 10),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
```

Comment:
You have used so much static values for size inside sizebox, It will create a problem while render on other resolutions.